x-pack/plugins/fleet/public/components/custom_assets_accordion.tsx

```diff
@@ -31,11 +31,13 @@ export interface CustomAssetsAccordionProps {
     description: string;
   }>;
   initialIsOpen?: boolean;
+  CSPDropdown?: string;
```

Comment:
`CSPDropdown` didn't sound much intuitive to me, it looks like it expects a Dropdown component or something like that, name, let's change to something more straightforward like `title`